tools/topology/CMakeLists.txt

```diff
@@ -92,9 +92,11 @@ set(TPLGS
 	"sof-apl-asrc-wm8804\;sof-apl-asrc-wm8804"
 	"sof-apl-asrc-pcm512x\;sof-apl-asrc-pcm512x"
 	"sof-apl-src-pcm512x\;sof-apl-src-pcm512x"
-	"sof-cml-rt5682\;sof-cml-rt5682\;-DPLATFORM=cml"
-	"sof-cml-rt5682\;sof-whl-rt5682\;-DPLATFORM=whl"
-	"sof-cml-rt5682\;sof-icl-rt5682\;-DPLATFORM=icl"
+	"sof-cml-rt5682\;sof-cml-rt5682\;-DPLATFORM=cml\;-DCPROC=volume"
```

Comment:
Maybe the need to specify volume could be avoided and define/default it in macro top level if missing? If volume needs to  be eliminated from pipeline -DxxxPROC could be set to passthrough to select pipe-passthrough-playback or capture.